modules/programs/eww.nix

```diff
@@ -23,12 +21,74 @@ in {
       '';
     };
 
-    configDir = mkOption {
-      type = types.path;
-      example = literalExpression "./eww-config-dir";
+    yuck = {
+      enable = mkEnableOption ""
+      mkOption {
```

Comment:
This looks like a mistake to me. Perhaps you mean to add a semicolon on the previous line and define a named option on this line?

Reply:
Ops a classic mistake that got missed.